sns.tf

```diff
@@ -16,3 +16,68 @@ resource "aws_sns_topic" "daily_registrations_reporter" {
   kms_master_key_id = aws_kms_alias.sns.arn
   tags              = module.labels.tags
 }
+
+data "aws_iam_policy_document" "sns_sms_logs_policy" {
+  statement {
+    actions   = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "logs:PutMetricFilter",
+      "logs:PutRetentionPolicy"]
+    resources = ["*"]
+  }
+}
+
+data "aws_iam_policy_document" "sns_sms_logs_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type = "Service"
+
+      identifiers = [
+        "sns.amazonaws.com"
+      ]
+    }
+  }
+}
+
+resource "aws_cloudwatch_log_group" "sns_sms_logs" {
+  count = var.enable_sms_publishing_with_aws ? 1 : 0
```

Comment:
I would create a local for this count rather than having to repeat the ? 1 : 0 in each of them